packages/babel-parser/src/parser/index.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import type { Options } from "../options";
-import type { File, JSXOpeningElement } from "../types";
+import type { File /*::, JSXOpeningElement */ } from "../types";
```

Comment:
This is needed otherwise ESLint marks it as unused (because it's "used" inside a comment).